src/config/config.ts

```diff
@@ -137,21 +149,27 @@ const config: AppIds = {
         title: 'ربما تم الكشف عن تداخل واحد أو أكثر',
         body: 'انقر هنا لمعرفة ما إذا كنت قد تعرضت'
       },
+      fr: {
+        title: 'Une ou plusieurs expositions peuvent avoir été détectées',
+        body: 'Cliquez ici pour savoir si vous avez été exposé'
+      },
       duration: 10000
     },
     furtherInstructions: {
       he: 'https://govextra.gov.il/ministry-of-health/corona/corona-virus/',
       en: 'https://govextra.gov.il/ministry-of-health/corona/corona-virus-en',
       am: 'https://govextra.gov.il/ministry-of-health/corona/corona-virus-am',
       ru: 'https://govextra.gov.il/ministry-of-health/corona/corona-virus-ru',
-      ar: 'https://govextra.gov.il/ministry-of-health/corona/corona-virus-ar'
+      ar: 'https://govextra.gov.il/ministry-of-health/corona/corona-virus-ar',
+      fr: 'https://govextra.gov.il/ministry-of-health/corona/corona-virus-en',
     },
     reportForm: {
       he: 'https://govforms.gov.il/mw/forms/QuarantineForExposees%40health.gov.il',
       en: 'https://govforms.gov.il/mw/forms/QuarantineForExposees%40health.gov.il',
       am: 'https://govforms.gov.il/mw/forms/QuarantineForExposees%40health.gov.il',
       ru: 'https://govforms.gov.il/mw/forms/QuarantineForExposees%40health.gov.il',
-      ar: 'https://govforms.gov.il/mw/forms/QuarantineForExposees%40health.gov.il'
+      ar: 'https://govforms.gov.il/mw/forms/QuarantineForExposees%40health.gov.il',
+      fr: 'https://govextra.gov.il/ministry-of-health/corona/corona-virus-en',
```

Comment:
same here, that's not the right link

Reply:
Same. 
No french on the ministry website so english is better than nothing